tools/integration-tests/tester/framework/parameters.go

```diff
@@ -124,6 +125,7 @@ type GoShimmerConfig struct {
 	WriteManaThreshold         float64
 	ReadManaThreshold          float64
 	SnapshotResetTime          bool
+	ForceCacheTime             int
```

Comment:
replace with time.Duration:

```suggestion
	ForceCacheTime             time.Duration
```